src/test/java/com/spring/boot/ApplicationTests.java

```diff
@@ -20,7 +28,21 @@ public void readCorrect() {
     }
 
     @Test
-    void contextLoads() {
+    void correctParseFromPathToObjects() {
+        ReviewDto reviewDto =new ReviewDto();
```

Comment:
```suggestion
        ReviewDto reviewDto = new ReviewDto();
```
pay attention to code style

Reply:
Done